src/patternfly/components/Chips/chip.scss

```diff
@@ -0,0 +1,185 @@
+  --pf-c-chip__text--Color: var(--pf-global--Color--100);
+  --pf-c-chip__text--MaxWidth: #{pf-size-prem(120px)}; // This can be changed for different use cases
+
+  // Chip button
+  --pf-c-chip__button--MinWidth: var(--pf-global--spacer--xl); // Change this to a new target size for accessibility
+  --pf-c-chip__button--MinHeight: #{pf-size-prem(28px)}; // Custom size for min height is as close as possible to target size for accessibility;
+  --pf-c-chip__button--PaddingRight: var(--pf-global--spacer--xs);
+  --pf-c-chip__button--PaddingLeft: var(--pf-global--spacer--xs);
+  --pf-c-chip__button--Color: var(--pf-global--Color--200);
+  --pf-c-chip__button--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-chip__button--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-chip__button--hover--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-chip__button--hover--BorderColor: var(--pf-global--BorderColor);
+  --pf-c-chip__button--hover--Color: var(--pf-global--primary-color--100);
+  --pf-c-chip__button--active--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-chip__button--active--BorderColor: var(--pf-global--BorderColor);
+  --pf-c-chip__button--active--Color: var(--pf-global--primary-color--200);
+  --pf-c-chip__button--focus--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-chip__button--focus--BorderColor: var(--pf-global--BorderColor);
+  --pf-c-chip__button--focus--Color: var(--pf-global--primary-color--200);
+
+  position: relative;
+  display: inline-flex;
+  padding-left: var(--pf-c-chip--PaddingLeft);
+  color: var(--pf-c-chip--Color);
+  vertical-align: middle;
+  background-color: var(--pf-c-chip--BackgroundColor);
+  border: 0;
+  border-radius: var(--pf-c-chip--BorderRadius);
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    border: var(--pf-c-chip--BorderWidth) solid var(--pf-c-chip--BorderColor);
+    border-radius: var(--pf-c-chip--BorderRadius);
+  }
+
+  &:not(:last-child) {
+    margin-right: var(--pf-c-chip--MarginRight);
+  }
+
+  // Chip Hover State
+  &.pf-m-hover,
+  &:hover {
+    .pf-c-button {
+      color: var(--pf-c-chip__button--hover--Color);
+    }
+
+    &::before {
+      border-color: var(--pf-c-chip--hover--BorderColor);
+    }
+  }
+
+  // Chip Overflow Modifier
+  &.pf-m-overflow {
+    padding-left: 0;
+    background-color: var(--pf-c-chip--m-overflow--BackgroundColor);
+
+    // Remove Chip border
+    &::before {
+      border: 0 transparent;
+    }
+
+    .pf-c-button {
+      padding-right: var(--pf-c-chip--m-overflow__button--PaddingRight);
+      padding-left: var(--pf-c-chip--m-overflow__button--PaddingLeft);
+
+      &.pf-m-active,
+      &:active::after {
+        border: var(--pf-c-chip__button--active--BorderWidth) solid var(--pf-c-chip__button--active--BorderColor);
+        border-radius: var(--pf-c-chip__button--BorderRadius);
```

Comment:
@mcoker I had to write it like this because the nested depth limitation was 3. I added "::after" to the end of "pf-m-active", i know this is a little inefficient, do you suggest anything else?

Reply:
Ah, in this case, it probably doesn't matter, but I'd argue we strive for consistency in formatting and just use `/* stylelint-disable */ ... /* stylelint-enable */` around that selector, or use `/* stylelint-disable-next-line */` if you can, to disable the linter for that line.

Reply:
thanks for the suggestion I used /* stylelint-disable-next-line */ for that line